packages/contracts/test/contracts/ovm/execution-manager/ExecutionManager.l1-l2-opcodes.spec.ts

```diff
@@ -133,17 +140,18 @@ describe('Execution Manager -- L1 <-> L2 Opcodes', () => {
       const passMessageToL1MethodId = bufToHexString(
         ethereumjsAbi.methodID('passMessageToL1', ['bytes'])
       )
-      const data = executionManager.interface.functions[
-        'executeTransaction'
-      ].encode([
-        getCurrentTime(),
-        0,
-        callContractAddress,
-        callBytes,
-        ZERO_ADDRESS,
-        ZERO_ADDRESS,
-        true,
-      ])
+      const data = executionManager.interface.encodeFunctionData(
```

Comment:
Am I correct that there's a "better" way to do this in Ethers 5? I vaguely remember that being the case

Reply:
Ah yeah I think this was it -- https://github.com/ethers-io/ethers.js/issues/535#issuecomment-498817969